src/Evolution/Initialization/ConservativeSystem.hpp

```diff
@@ -20,8 +32,9 @@ struct Inertial;
 /// \endcond
 
 namespace Initialization {
-
-/// \brief Allocate variables needed for evolution of conservative systems
+namespace Actions {
```

Comment:
I suggest to remove the outer `Initialization` namespace and instead place the evolution-specific initializers in `Evolution::Actions`, e.g. `Evolution::Actions::InitializeConservativeSystem`. Reason is that initializers are regular actions now, so we should them as such. Also, actions should have verbs for names.

Reply:
Meh, I'm not sure about the evolution renaming. I'm not for or against, I'd rather leave it as is.

As I've said before, I'm not "namespacing" all the initialization actions with `Initialize`. They should just go in an `Initialization` namespace.

Reply:
Then, how about we swap the order of nested namespaces: `Actions::Initialize::ConservativeSystem` or `Actions::Initialization::ConservativeSystem`
Alternatively, we could skip the `Actions` namespace for initializers and do `Initializers::ConservativeSystem` or `InitializeActions::ConservativeSystem`

Reply:
we decided in a call to do names like `Actions::InitializeDomain`

Reply:
In a future PR :)